geospatial/dted.ksy

```diff
@@ -0,0 +1,596 @@
+          Longitude of SW corner of data, bounding rectangle— leading zeroes |
+          for values less than 100; H is the hemisphere of the data (DDDMMSSH).
+      - id:  nw_corner_of_data
+        type: coordinate_pair_ddmmssh
+        doc: |
+          Latitude of NW corner of data, bounding rectangle— leading zero for |
+          values less than 10; H is the hemisphere of the data. (DDMMSSH) |
+          Longitude of NW corner of data, bounding rectangle— leading zeroes |
+          for values less than 100; H is the hemisphere of the data (DDDMMSSH).
+      - id:  ne_corner_of_data
+        type: coordinate_pair_ddmmssh
+        doc: |
+          Latitude of NE corner of data, bounding rectangle—leading zero for |
+          values less than 10; H is the hemisphere of the data (DDMMSSH). |
+          Longitude of NE corner of data, bounding rectangle— leading zeroes |
+          for values less than 100; H is the hemisphere of the data (DDDMMSSH).
+      - id:  se_corner_of_data
+        type: coordinate_pair_ddmmssh
+        doc: |
+          Latitude of SE corner of data, bounding rectangle—leading zero for |
+          values less than 10; H is the hemisphere of the data.
+          Longitude of SE corner of data, bounding rectangle— leading zeroes |
+          for values less than 100; H is the hemisphere of the data.
+      - id:  clockwise_orientation_angle
+        type: angle_dddmmss_sh
+        doc: |
+          Clockwise orientation angle of data with respect to true North. |
+          (Will usually be all zeros.)
+      - id:  latitude_interval
+        type: ascii_int
+        doc: |
+          Latitude interval in tenths of seconds between rows of elevation |
+          values (SSSS).
+      - id:  longitude_interval
+        type: ascii_int
+        doc: |
+          Longitude interval in tenths of seconds between columns of elevation |
+          values (SSSS).
+      - id:  number_of_latitude_lines
+        type: ascii_int
+        doc: |
+          Number of Latitude lines. For magnetic tape, this is the actual |
+          count of the number of latitude points (rows that contain data). |
+          For CD-ROM, this is the count of the number of latitude points in |
+          a full one-degree cell. (e.g. 1201 for DTED1, 3601 for DTED2) |
+          (0000-9999).
+      - id:  number_of_longitude_lines
+        type: ascii_int
+        doc: |
+          Number of Longitude lines. For magnetic tape, this is the actual |
+          count of the number of longitude points (columns that contain data). |
+          For CD-ROM, this is the count of the number of longitude points in a |
+          full one-degree cell. The count is based on the level of DTED and |
+          the latitude zone of the cell. (See Table II and III) (0000-9999). 
+      - id:  partial_cell_indicator
+        type: str
+        size: 2
+        doc: |
+          Partial Cell Indicator 
```

Comment:
I propose to rename it and create an instance getting actual coverage.

Reply:
Ok I'll rename the original field to partial_cell_indicator_text and add a calculated instance field which looks as follows
```
    instances:
      partial_cell_indicator:
        value: (partial_cell_indicator_text.to_i == 0) ? | 100.0 : partial_cell_indicator_text.to_i
        doc: |
          Partial Cell Indicator (% of data coverage)
```

Reply:
`partial_cell_indicator_text.to_i` is calculated twice. It may make sense to create an instance for that.